### ISSUE TITLE
`-o` option not supported in `cmds/core/ip` cmd.

### ISSUE THREAD

**Issue by 10000TB** [u-root/u-root#3195]
```
➜ ./ip Comment start  -ao Comment end  addr show 

2024/11/14 12:24:51 ip: outputting each record on a single line is unsupported 

```

I was wondering IF this option is not supported, any reason why we still keep it as part of help message ?

@RiSKeD / @jensdrenhaus shared there was challenges in implementing in the MVP phase.

**Comment by 10000TB** [comment]
@RiSKeD could you elaborate the challenges / considerations that went in the decision on not implementing  this as part of MVP delivery ?

**Comment by RiSKeD** [comment]
Of course, this option was dropped as priority was shifted towards more crucial flags inside the IP command. There are no challenges per se implementation-wise, just more of a prioritization for the MVP delivery.

**Comment by jensdrenhaus** [comment]
This is a low-hanging fruit to add on the way forward.
